src/dictionary/manager.ts

```diff
@@ -171,6 +171,16 @@ class SchemaManager {
     return result;
   };
 
+  fetchDiff = async (oldVersion: string, newVersion: string) => {
```

Comment:
Is this still needed ?

Reply:
it's not used right now, but I thought maybe it can be used in the future so I kept it